frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -59,8 +59,15 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
               }
             </Nav>
 
-
-
+            <Nav className="mr-auto">
+            {
+              hasRole(currentUser, "ROLE_ADMIN") && (
```

Comment:
Why only ROLE_ADMIN when the App.js says ROLE_USER?

I think probably this should be unprotected and all users should be able to access it, since it doesn't update any state on the server, and is just looking up publicly available info.

But at least be consistent; I suggest both should be ROLE_USER if you are going to restrict access.